src/containers/confirm-audio-to-waudio/desktop/ConfirmAudioToWAudioModal.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useCallback, useState } from 'react'
+
+import { Button, ButtonType, Modal } from '@audius/stems'
+import cn from 'classnames'
+import { useDispatch } from 'react-redux'
+
+import Drawer from 'components/drawer/Drawer'
+import LoadingSpinner from 'components/loading-spinner/LoadingSpinner'
+import { getModalVisibility } from 'store/application/ui/modals/slice'
+import { confirmTransferAudioToWAudio } from 'store/audio-manager/slice'
+import { isMobile as checkIsMobile } from 'utils/clientUtil'
+import { useSelector } from 'utils/reducer'
+
+import styles from './ConfirmAudioToWAudioModal.module.css'
+
+const messages = {
+  title: 'Hang Tight ',
+  header: 'Audius is now much faster!',
+  description:
+    'Before you can start listening we’ll have to upgrade your $AUDIO.',
+  moreInfo: 'More Info',
+  confirm: 'Let’s Do It!',
+  loadingTitle: '✋ Hold on, this will just take a moment.',
```

Comment:
@raymondjacobson where do we get these icons from? I should probably replace this